README.md

```diff
@@ -337,329 +337,3 @@ Introduction to the Splunk product and some of its capabilities
 ## Contact
 
 You can reach the Dev Platform team at [devinfo@splunk.com](mailto:devinfo@splunk.com).

```

Comment:
@afinkenstadt Can you refresh against master? The PR shouldn't show removing all of the content below if you are just making slight updates.